remo/events/forms.py

```diff
@@ -299,6 +298,10 @@ def _clean_bug(self, bug_id):
         bug, created = Bug.objects.get_or_create(bug_id=bug_id)
         return bug
 
+    def clean_categories(self):
+        category_id = self.cleaned_data['categories']
+        return get_object_or_none(FunctionalArea, id=category_id)
```

Comment:
This is an edge case (and requires client side tampering from the user) but if ``clean_categories`` returns ``None`` we might end up with a required field saved without data.